### PR TITLE
Test

### PR DESCRIPTION
**Pull request by sepandhaghighi**
Tests system added

### CODE REVIEW

File: nafas/__main__.py [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
-    input_data = get_input_standard()
-    filtered_data = input_filter(input_data)
-    program_data = get_program_dict(filtered_data)
-    run(program_data)
\ No newline at end of file
+    if len(args) < 1:
+        input_data = get_input_standard()
+        filtered_data = input_filter(input_data)
+        program_data = get_program_dict(filtered_data)
+        run(program_data)
```
Comment (sadrasabouri) [inline nafas/__main__.py:18]: A new line here would be proper.

**Review by sadrasabouri — COMMENTED** [review]
Good job.
I've reviewed it and there was a new line miss in one of `.py` files.

File: nafas/__main__.py (hunk 1 above)
Comment (sepandhaghighi) [reply nafas/__main__.py]: Thanks ;-)
Solved in https://github.com/sepandhaghighi/nafas/pull/9/commits/bf512418224172d4281fc3594cd569a567f33039

**Review by sadrasabouri — APPROVED** [review]
Thank you.

**Pull request merged**